versions/baseline.json

```
@@ -8909,4 +8913,4 @@
       "port-version": 3
     }
   }
-}
+}
```

```suggestion
}

```
Adding a blank line at the end is a formatting requirement (there is a blank line, you accidentally changed it), I tried to submit but I don't have permission.
`fatal: unable to access 'https://github.com/marcodiiga/vcpkg.git/': The requested URL returned error: 403`